src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -1878,6 +1881,8 @@ public override DeclarationKind GetDeclarationKind(SyntaxNode declaration)
             {
                 case SyntaxKind.ClassDeclaration:
                     return DeclarationKind.Class;
+                case SyntaxKind.RecordDeclaration:
+                    return DeclarationKind.Record;
```

Comment:
This has an error: `'DeclarationKind' does not contain a definition for 'Record'`

Searching through the code, there are:

- A `DeclarationKind` enum in `Microsoft.CodeAnalysis.CSharp` namespace, which contain `Record` member.
- A `DeclarationKind` enum in `Microsoft.CodeAnalysis.Editing` namespace, which doesn't contain `Record` member.

I'm not sure what to do here?

Should I just return DeclarationKind.Class for both classes and records?

Note that this public method:

https://docs.microsoft.com/en-us/dotnet/api/microsoft.codeanalysis.editing.syntaxgenerator.getdeclarationkind?view=roslyn-dotnet

is dependent on this code path.

Reply:
> A DeclarationKind enum in Microsoft.CodeAnalysis.Editing namespace, which doesn't contain Record member.

Fix this up.

Reply:
This being fixed in #48096